ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-ec-4.13.yaml

```diff
@@ -159,6 +163,17 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test-disconnected
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private
+- as: aws-ipi-ovn-hypershift-p1-f7
+  cron: 44 1 2,10,18,25 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      SKIP_HYPERSHIFT_PULL_SECRET_UPDATE: "true"
```

Comment:
I think we should add some hypershift tag to filter test cases that are applicable to it. @kuiwang02 has better insight on the tag.

Reply:
@kuiwang02  is this right?
```
TEST_FILTERS: "~DisconnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&;~NonHyperShiftHOST&"
```

Reply:
@LiangquanLi930 it is correct.